includes/class-wc-gateway-stripe.php

```diff
@@ -347,72 +335,17 @@ public function elements_form() {
 		<?php
 	}
 
-	/**
-	 * Output the React Settings page wrapper
-	 */
-	private function output_settings_page_wrapper() {
-		global $hide_save_button;
-		$hide_save_button = true;
-		echo '<h2>' . esc_html( $this->get_method_title() );
-		wc_back_link( __( 'Return to payments', 'woocommerce-gateway-stripe' ), admin_url( 'admin.php?page=wc-settings&tab=checkout' ) );
-		echo '</h2>';
-		echo '<div id="wc-stripe-account-settings-container"></div>';
-	}
-
 	/**
 	 * Maybe override the parent admin_options method.
 	 */
 	public function admin_options() {
-		if ( ! WC_Stripe_Feature_Flags::is_upe_enabled() ) {
+		if ( ! WC_Stripe_Feature_Flags::is_upe_settings_redesign_enabled() ) {
 			parent::admin_options();
-			return;
-		}
-		$this->output_settings_page_wrapper();
-	}
 
-	/**
-	 * Load admin scripts.
-	 *
-	 * @since   3.1.0
-	 * @version 3.1.0
-	 */
-	public function admin_scripts() {
```

Comment:
It looks like this was duplicated between the `WC_Stripe_UPE_Payment_Gateway` and the `WC_Gateway_Stripe` class - refactored and moved to the `WC_Stripe_Settings_Controller` class so that they're both using the same logic to enqueue the scripts.